src/freeze-dry/remove-scripts.test.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-env jest */
+
+import removeScripts from 'src/freeze-dry/remove-scripts'
+
+describe('removeScripts', () => {
+    test('should remove script tags from the document', async () => {
+        const parser = new DOMParser()
+        const rootElement = parser.parseFromString(
+            '<html><head><script></script></head></html>',
+            'text/html'
+        )
+        const docUrl = 'about:blank'
+        await removeScripts({rootElement, docUrl})
+        expect(rootElement.getElementsByTagName('script').length).toBe(0)
+    })
+
+    test('should remove "on" handlers', async () => {
+        const parser = new DOMParser()
+        const rootElement = parser.parseFromString(
+            '<html><div onHover="handler()" onClick="handler()"></div></html>',
```

Comment:
Not a big problem, but `onHover` or `onhover` does not exist. `onmouseover` does. `onclick` is lowercase.